CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.0.0-beta.1
+- First beta for RevenueCat (previously Purchases) framework 🎉
+    100% Swift framework + ObjC support.
+    
```

Comment:
can we provide either an early migration guide, or some list of the changes that folks should expect so far? 
Also, I think we should mention the SPM thing as a known issue

Reply:
I was holding off for a later release since I'm sure we'll spin another build very soon, but you're totally right, no need to wait and I might as well do it now.

Reply:
https://github.com/RevenueCat/purchases-ios/issues/843 to track fastlane doc generation.